dbt/context/common.py

```diff
@@ -148,8 +148,12 @@ def _add_sql_handlers(context):
     })
 
 
-def log(msg):
-    logger.debug(msg)
+def log(msg, info=False):
+    if info:
+        logger.info(" ↳ " + msg)
```

Comment:
@cmcarthur i really like this but fear that it will randomly break python2 installations. Will remove

Reply:
oh lol yeah it broke circle